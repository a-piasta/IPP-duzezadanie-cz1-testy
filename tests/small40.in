extendRoute;50;hxaxfxrzmivjoji
repairRoad;hxaxfxrzmivjoji;ryafoqcewtjl;-2
repairRoad;cxcefjblemqko;gvlfdzgbyrziapqmiqwxkzg;-5
newRoute;165;uqkuhcnfyyq;ryafoqcewtjl
addRoad;gvlfdzgbyrziapqmiqwxkzg;nypptyujjhhfvqhwrunrwvfmzmis;10;4
repairRoad;uiyfjxqrszpoyioz;hxaxfxrzmivjoji;-13
getRouteDescription;165
repairRoad;uiyfjxqrszpoyioz;uiyfjxqrszpoyioz;-9
repairRoad;uqkuhcnfyyq;cxcefjblemqko;1
extendRoute;165;hxaxfxrzmivjoji
newRoute;859;hxaxfxrzmivjoji;cxcefjblemqko
repairRoad;sazoqkkamx;sazoqkkamx;3
getRouteDescription;859
getRouteDescription;859
repairRoad;nypptyujjhhfvqhwrunrwvfmzmis;zjxhjoimrxinuchcdpnche;-10
addRoad;gvlfdzgbyrziapqmiqwxkzg;ryafoqcewtjl;18;-13
newRoute;210;nypptyujjhhfvqhwrunrwvfmzmis;zjxhjoimrxinuchcdpnche
getRouteDescription;165
addRoad;nypptyujjhhfvqhwrunrwvfmzmis;uiyfjxqrszpoyioz;4;-2
newRoute;395;cxcefjblemqko;gvlfdzgbyrziapqmiqwxkzg
newRoute;639;uiyfjxqrszpoyioz;cxcefjblemqko
repairRoad;gvlfdzgbyrziapqmiqwxkzg;zjxhjoimrxinuchcdpnche;13
repairRoad;sazoqkkamx;nooowqfpfnfbmratgeqllvumjabz;-9
newRoute;63;gvlfdzgbyrziapqmiqwxkzg;hxaxfxrzmivjoji
removeRoad;nypptyujjhhfvqhwrunrwvfmzmis;uiyfjxqrszpoyioz
newRoute;776;cxcefjblemqko;gvlfdzgbyrziapqmiqwxkzg
getRouteDescription;63
addRoad;ryafoqcewtjl;ryafoqcewtjl;14;11
newRoute;311;nypptyujjhhfvqhwrunrwvfmzmis;ryafoqcewtjl
extendRoute;165;uiyfjxqrszpoyioz
extendRoute;50;hxaxfxrzmivjoji
repairRoad;ryafoqcewtjl;ryafoqcewtjl;-6
removeRoad;nypptyujjhhfvqhwrunrwvfmzmis;gvlfdzgbyrziapqmiqwxkzg
addRoad;zjxhjoimrxinuchcdpnche;nooowqfpfnfbmratgeqllvumjabz;14;9
removeRoad;gvlfdzgbyrziapqmiqwxkzg;nypptyujjhhfvqhwrunrwvfmzmis
removeRoad;nypptyujjhhfvqhwrunrwvfmzmis;uiyfjxqrszpoyioz
removeRoad;gvlfdzgbyrziapqmiqwxkzg;ryafoqcewtjl
addRoad;nooowqfpfnfbmratgeqllvumjabz;uiyfjxqrszpoyioz;15;-11
getRouteDescription;639
removeRoad;ryafoqcewtjl;gvlfdzgbyrziapqmiqwxkzg
getRouteDescription;311
repairRoad;hxaxfxrzmivjoji;nooowqfpfnfbmratgeqllvumjabz;-13
addRoad;hxaxfxrzmivjoji;sazoqkkamx;22;-6
extendRoute;63;nypptyujjhhfvqhwrunrwvfmzmis
addRoad;gvlfdzgbyrziapqmiqwxkzg;nypptyujjhhfvqhwrunrwvfmzmis;28;-15
repairRoad;uiyfjxqrszpoyioz;sazoqkkamx;1
removeRoad;nypptyujjhhfvqhwrunrwvfmzmis;gvlfdzgbyrziapqmiqwxkzg
newRoute;704;sazoqkkamx;sazoqkkamx
removeRoad;gvlfdzgbyrziapqmiqwxkzg;nypptyujjhhfvqhwrunrwvfmzmis
newRoute;462;hxaxfxrzmivjoji;zjxhjoimrxinuchcdpnche
removeRoad;hxaxfxrzmivjoji;sazoqkkamx
getRouteDescription;395
repairRoad;cxcefjblemqko;sazoqkkamx;-4
getRouteDescription;268
newRoute;559;cxcefjblemqko;ryafoqcewtjl
addRoad;nooowqfpfnfbmratgeqllvumjabz;cxcefjblemqko;14;-10
repairRoad;nooowqfpfnfbmratgeqllvumjabz;nypptyujjhhfvqhwrunrwvfmzmis;-14
extendRoute;462;ryafoqcewtjl
