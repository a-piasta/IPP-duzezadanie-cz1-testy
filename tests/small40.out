1: NIE
2: NIE
3: NIE
4: NIE
5: TAK
6: NIE
7: 
8: NIE
9: NIE
10: NIE
11: NIE
12: NIE
13: 
14: 
15: NIE
16: TAK
17: NIE
18: 
19: TAK
20: NIE
21: NIE
22: NIE
23: NIE
24: NIE
25: TAK
26: NIE
27: 
28: NIE
29: TAK
30: NIE
31: NIE
32: NIE
33: NIE
34: TAK
35: NIE
36: NIE
37: NIE
38: TAK
39: 
40: NIE
41: 311;nypptyujjhhfvqhwrunrwvfmzmis;10;4;gvlfdzgbyrziapqmiqwxkzg;18;-13;ryafoqcewtjl
42: NIE
43: TAK
44: NIE
45: NIE
46: NIE
47: NIE
48: NIE
49: NIE
50: NIE
51: TAK
52: 
53: NIE
54: 
55: NIE
56: TAK
57: NIE
58: NIE
